calculator.Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Build the manager binary
+FROM golang:1.16 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY cmd/size-calculator/main.go main.go
+COPY api/ api/
```

Comment:
Do you need the operator api here?

Reply:
Just needed for using the size constants. I have removed that and added it to same file so that it has no dependency.